src/main/java/org/jenkinsci/test/acceptance/plugins/external_workspace_manager/ExternalGlobalConfig.java

```diff
@@ -0,0 +1,33 @@
+package org.jenkinsci.test.acceptance.plugins.external_workspace_manager;
+
+import org.jenkinsci.test.acceptance.po.PageAreaImpl;
+import org.jenkinsci.test.acceptance.po.PageObject;
+
+/**
+ * Helper class for interacting with External Workspace Manager Plugin global config page.
+ *
+ * @author Alexandru Somai
+ */
+public class ExternalGlobalConfig extends PageAreaImpl {
+
+    public ExternalGlobalConfig(PageObject context) {
```

Comment:
It is a good practice to require the exact type of PageObject in PageArea so it is obvious to the client how to use it.